static/js/tippyjs.js

```diff
@@ -179,4 +179,14 @@ export function initialize() {
             instance.destroy();
         },
     });
+
+    delegate("body", {
+        target: ".in_message_time",
+        allowHTML: true,
+        placement: "right",
+        appendTo: () => document.body,
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
```

Comment:
It would've been great to avoid this `delegate` call but the `appendTo: () => document.body` arg is important as appending it to `parent` would only render a part of the tooltip.